README.md

```diff
@@ -1,3 +1,252 @@
+
+Install prerequisites:
+
+```
+pip install cookiecutter
+```
+
+In the root of the repo, create a new project:
+
+```bash
+cookiecutter templates/basic -o projects
+```
+
+You'll need to answer two questions:
+```
+project_name: my_project  # replace this with your project name
+project_author: foobar    # replace this with your name
+```
+
+> **NOTE:** For [MLOps Community Engineering Labs Hackathon](https://flyte.org/hackathon/) participants:
+> `project_author` should be your team name.
+
+The project structure looks like the following:
+```bash
+.
+├── Dockerfile
+├── README.md
+├── dashboard
+│   ├── app.py  # streamlit app
+│   ├── remote.config
+│   └── sandbox.config
+├── deploy.py  # deployment script
+├── my_project
+│   ├── __init__.py
+│   └── workflows.py  # flyte workflows
+├── requirements-dev.txt
+└── requirements.txt
+```
+
+## 🌏 Environment Setup
+
+Go into the project directory, then create your project's virtual environment:
+
+```bash
+cd projects/my_project
+
+# create and activate virtual environment
+python -m venv env
+source env/bin/activate
+
+# install requirements
+pip install -r requirements.txt -r requirements-dev.txt
+```
+
+Run Flyte workflows locally:
+
+```
+python my_project/workflows.py
+```
+
+You should see something like this in the output (you can ignore the warnings):
+```
+trained model: LogisticRegression()
+```
+
+Congrats! You just setup your flytelab project 🌟. You can now modify and iterate on the
+`workflows.py` file to create your very own Flyte workflows using `flytekit`. You can refer to the
+[User Guide](https://doc s.flyte.org/projects/cookbook/en/latest/index.html),
+[Tutorials](https://docs.flyte.org/projects/cookbook/en/latest/tutorials.html),
+and [Flytekit API Reference](https://docs.flyte.org/projects/flytekit/en/latest/) to
+learn more about all of `Flyte`'s capabilities.
+
+## 🚢 Deployment
+
+So far you've probably been running your workflows locally by invoking `python my_project/workflows.py`.
+The first step to deploying your workflows to a Flyte cluster is to test it out on a
+[local sandbox cluster](https://docs.flyte.org/en/latest/deployment/sandbox.html).
+
+Make sure you have [docker](https://docs.docker.com/get-docker/) installed.
+
+Then install `flytectl`:
+
+<details>
+
+<summary>OSX</summary>
+
+```bash
+brew install flyteorg/homebrew-tap/flytectl
+```
+
+</details>
+
+<details>
+
+<summary>Other Operating Systems</summary>
+
+```bash
+curl -sL https://ctl.flyte.org/install | sudo bash -s -- -b /usr/local/bin # You can change path from /usr/local/bin to any file system path
+export PATH=$(pwd)/bin:$PATH # Only required if user used different path then /usr/local/bin
+```
+
+</details>
+
+### Sandbox Deployment
+
+Start the sandbox cluster:
+
+```bash
+flytectl sandbox start --source .
+```
+
+You should now be able to go to `http://localhost:30081/console` on your browser to see the Flyte UI.
+
```

Comment:
Do we have to ask the user to commit changes to a PR before running `python deploy.py`?